scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -951,18 +1193,182 @@ def _generate_tmcf(self, df_cols: list, name: str) -> None:
+                    if check_url.status_code == 200:
+                        _FILES_TO_DOWNLOAD.append(
+                            {"download_path": url_to_check})
+
+                except:
+                    logging.error(f"URL is not accessable {url_to_check}")
+
+
+def download_files():
+    """
+    This method allows to download the input files.
+
+    """
+
+    global _FILES_TO_DOWNLOAD
+    session = requests.session()
+    max_retry = 5
+    for file_to_dowload in _FILES_TO_DOWNLOAD:
+        file_name = None
+        download_local_path = _INPUT_FILE_PATH
+        _url = file_to_dowload['download_path']
+        if 'file_name' in file_to_dowload and len(
+                file_to_dowload['file_name'] > 5):
+            file_name = file_to_dowload['file_name']
+        else:
+            file_name = _url.split('/')[-1]
+        retry_number = 0
+
+        is_file_downloaded = False
+        while is_file_downloaded == False:
+            try:
+                if ".csv" in _url:
+                    if "st-est" in _url or 'SC-EST' in _url:
+                        file_name = file_name.replace(".csv", ".xlsx")
+                        df = pd.read_csv(_url, on_bad_lines='skip', header=0)
+                        df.to_excel(download_local_path + os.sep + file_name\
+                            ,index=False,engine='xlsxwriter')
+                    elif "pe-11" in _url:
+                        df = _resolve_pe_11(file_name, _url)
+                        df.to_csv(download_local_path + os.sep + file_name,
+                                  index=False)
+                    elif "pe-19" in _url:
+                        file_name = file_name.replace(".csv", ".xlsx")
+                        df = pd.read_csv(_url,
+                                         skiprows=5,
+                                         on_bad_lines='skip',
+                                         header=0)
+                        df.to_excel(download_local_path + os.sep + file_name\
+                            ,index=False,engine='xlsxwriter')
+                    elif "co-asr-7079" in _url or "pe-02" in _url:
+                        file_name = file_name.replace(".csv", ".xlsx")
+                        cols=['Year','FIPS','Race/Sex',1,2,3,4,5,6,7,8,9,10,11,12,\
+                            13,14,15,16,17,18]
+                        if "pe-02" in _url:
+                            df = pd.read_csv(_url, skiprows=7, on_bad_lines='skip', \
+                                names=cols)
+                        else:
+                            df = pd.read_csv(_url,
+                                             on_bad_lines='skip',
+                                             names=cols)
+                        df.to_excel(download_local_path + os.sep + file_name,\
+                            index=False,engine='xlsxwriter')
+                    elif "co-est00int-alldata" in _url or "CC-EST2020-ALLDATA" in _url or "cc-est2022-all":
```

Comment:
` or "cc-est2022-all":` will cause the condition to always succeed.

should it be:
`elif "co-est00int-alldata" in _url or "CC-EST2020-ALLDATA" in _url or "cc-est2022-all" in _url:`

Reply:
yes missed you add "cc-est2022-all" in _url now its modified